pandas/core/groupby/generic.py

```diff
@@ -2044,6 +2044,7 @@ def _gotitem(self, key, ndim: int, subset=None):
         elif ndim == 1:
             if subset is None:
                 subset = self.obj[key]
+
```

Comment:
Can you revert this line addition

Reply:
resolved

Reply:
This still appears in the diff of this PR.